var/spack/repos/builtin/packages/cubist/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Cubist(Package):
```

Comment:
Can you convert this to a `MakefilePackage`? Then `patch(self)` will become `edit(self, spec, prefix)` and `make()` can be removed from the `install` method.

Reply:
done